api/resource_groups_gcp.go

```diff
@@ -0,0 +1,115 @@
+//
+// Author:: Darren Murray (<darren.murray@lacework.net>)
+// Copyright:: Copyright 2021, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package api
+
+import (
+	"encoding/json"
+	"strconv"
+
+	"github.com/pkg/errors"
+)
+
+// Wildcard values for selecting all gcp projects
+var (
+	GcpResourceGroupAllProjects = []string{"*"}
+)
+
+// GetGcpResourceGroup gets a single Gcp ResourceGroup matching the
+// provided resource guid
+func (svc *ResourceGroupsService) GetGcpResourceGroup(guid string) (
+	response GcpResourceGroupResponse,
+	err error,
+) {
+	var rawResponse resourceGroupWorkaroundResponse
+	err = svc.get(guid, &rawResponse)
+	if err != nil {
+		return
+	}
+
+	return setGcpResponse(rawResponse)
+}
+
+// UpdateGcpResourceGroup updates a single Gcp ResourceGroup on the Lacework Server
+func (svc *ResourceGroupsService) UpdateGcpResourceGroup(data ResourceGroup) (
+	response GcpResourceGroupResponse,
+	err error,
+) {
+	var rawResponse resourceGroupWorkaroundResponse
+	err = svc.update(data.ID(), data, &rawResponse)
+	if err != nil {
+		return
+	}
+
+	return setGcpResponse(rawResponse)
+}
+
+func setGcpResponse(response resourceGroupWorkaroundResponse) (gcp GcpResourceGroupResponse, err error) {
+	var props GcpResourceGroupProps
+
+	isDefault, err := strconv.Atoi(response.Data.IsDefault)
+	if err != nil {
+		return GcpResourceGroupResponse{}, err
```

Comment:
Nit> again no need to pass variables here
```suggestion
		return
```